src/pages/Details/components/CartModal.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useRef } from 'react';
+import './CartModal.scss';
+import Button from './Button';
+import { Link, useNavigate } from 'react-router-dom';
+
+const CartModal = props => {
+  const { setModalOpen, title } = props;
```

Comment:
```suggestion
const CartModal = ({ setModalOpen, title }) => {
```

위와 같이 구조분해 할당 적용해주세요!